modules/migration-upgrading-migration-tool-4.adoc

```diff
@@ -21,27 +20,42 @@ The following procedure enables you to change the *Manual* approval option to *A
 . Optional: Edit the *Channel*.
 +
 Updating the subscription deploys the updated {mtc-full} Operator and updates the {mtc-short} components.
-endif::[]
-ifeval::["{mtc-version}" <= "1.3"]
-You can upgrade to {mtc-full} ({mtc-short}) {mtc-version} on an {product-title} 4 cluster by deleting the `MigrationController` custom resource (CR), uninstalling the CAM Operator, and then installing the {mtc-full} Operator.
 
-.Procedure
+. Click *{mtc-full} Operator*.
```

Comment:
As the instruction is to describe the MTC upgrade from 1.3.x -> 1.4.0, step #5,6,7 should be removed.

Reply:
Are you saying that the migration controller parameters are preserved? In other words, the user has to re-create the migration controller but does not need to reset the parameters in the manifest?

Reply:
yes, it is on 4.4, 4.7

Reply:
OK. I'll fix that.